src/render.js

```diff
@@ -648,16 +648,20 @@ function InitRender(deps) {
       formElement.removeClass('loading').addClass('success');
 
       showIcsButtons(formElement, formData.email, eventData);
-
-    }).catch(function(){
-
-      showBookingFailed(formElement)
-
+    }).catch(function(err){
+      showBookingFailed(formElement, err)
     });
-
   };
 
-  var showBookingFailed = function (formElement) {
+  var showBookingFailed = function (formElement, err) {
+    if (err) {
+      if(err.status === 422 && err.data && err.data.errors['customer.email']) {
+        if ($('.email-error').length === 0) {
+          formElement.find('.input-email').after('<div class="email-error">Please enter a valid email.</div>');
+          formElement.find('.input-email').addClass('error');
```

Comment:
You don't need to do the find twice. You can either assign the input to a variable and use that variable twice, or just chain everything together like so:
```
formElement.find('.input-email')
    .addClass('error')
    .after('<div class="email-error">Please enter a valid email.</div>');
```